cmd/auth-vue/src/assets/.gitkeep

```diff
@@ -0,0 +1,5 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Not sure if it's really important, but there's a "no newline" warning here showing up in the GitHub diff UI

Reply:
it shows up at the end of files.

It's actually good practice to always leave an **empty** line at the end of files. This way the diff is a bit less confusing if you happen to change the last line with contents